docs/section-6-contributing.md

```diff
@@ -143,3 +143,43 @@ Publishing a new release of the CLI requires these steps:
    cd cli/npm
    npm publish
    ```
+
+## Developing Individual Language Grammars
+
+Grammars for individual languages are developed in `tree-sitter/tree-sitter-LANGUAGE` repositories. This section describes general practices for language grammars.
```

Comment:
The question is perhaps what we think the scope of these guidelines are. It would be nice of grammars outside the Tree-sitter organization follow the same guidelines, but that is perhaps harder to influence? The language here could be changed to make clear that this is the Tree-sitter org convention, and that this is recommended for all other grammar development as well?